xml/System.Net/HttpWebRequest.xml

```diff
@@ -1899,14 +1899,7 @@ Both <xref:System.Net.HttpWebRequest> constructors are obsolete and should not b
 
 [!INCLUDE [obsolete-notice](~/includes/web-request-obsolete.md)]
 
- The <xref:System.Net.HttpWebRequest.ConnectionGroupName%2A> property enables you to associate a request with a connection group. This is useful when your application makes requests to one server for different users, such as a Web site that retrieves customer information from a database server.
-
-## Examples
```

Comment:
Why are we deleting this example? If it really should be deleted, shouldn't the snippet files also go?

Reply:
The tool found a lot of orphaned snippets, so I'll clean them up in a separate PR instead of dirtying this one.